.github/workflows/bumpversion.yml

```diff
@@ -0,0 +1,36 @@
+name: Bump_Version
+
+on:
+  repository_dispatch:
+    types: [kolibri-explore-plugin]
```

Comment:
I think the type should probably called `kolibri-explore-plugin-release` just to be a bit more specific than "something happened on kolibri-explore-plugin".

Reply:
ok!

Reply:
Updated by the new commit.